src/App/Pages/Settings/SecuritySettingsPageViewModel.cs

```diff
@@ -76,7 +78,7 @@ public SecuritySettingsPageViewModel()
                 _logger,
                 OnVaultTimeoutActionChangingAsync,
                 AppResources.SessionTimeoutAction,
-                _ => _inited && !HasVaultTimeoutActionPolicy,
+                _ => _inited && !HasVaultTimeoutActionPolicy && IsVaultTimeoutActionLockAllowed,
```

Comment:
🎨 Given that now the action is enabled based on `IsvaultTimeoutActionLockAllowed`, you need to call `VaultTimeoutActionPickerViewModel.SelectOptionCommand.RaiseCanExecuteChanged();` when that changes so it gets enabled/disabled accordingly.